spring-integration-file/src/main/resources/org/springframework/integration/file/config/spring-integration-file-4.3.xsd

```diff
@@ -509,6 +509,17 @@ Only files matching this regular expression will be picked up by this adapter.
 				</xsd:appinfo>
 			</xsd:annotation>
 		</xsd:attribute>
+		<xsd:attribute name="preserve-timestamp" type="xsd:string" default="false">
+			<xsd:annotation>
+				<xsd:documentation>
+					Specify whether to preserve the modified timestamp from the source
+					file on the local file after copying. Applies to 'java.io.File' payloads.
```

Comment:
:+1: 

Right, I see that in the Docs. But that JavaDoc should be fixed for this as well.